### DISCUSSION TITLE
Is uint missing on purpose?

### DISCUSSION THREAD

**Discussion by marcovc** [hsutter/cppfront#752] [Q&A]
Hi!

While browsing this part of the util header,

//  Discouraged: Variable precision names 
 //                 short 
 using ushort     = unsigned short; 
 //                 int 
 using ulong      = unsigned long; 
 //                 long 
 using longlong   = long long; 
 using ulonglong  = unsigned long long; 
 using longdouble = long double;

I noticed there is no uint. Is this on purpose?

**Comment by SRNissen** [comment]
Surprisingly, `uint` is not a C++ integer type.

https://en.cppreference.com/w/cpp/language/types#Integer_types

**Reply by marcovc** [reply]
What do you mean? "unsigned int" is in that table, right?

**Reply by SRNissen** [reply]
My apologies, I misunderstood your question - you are of course right that cpp1 does have unsigned integers.

I thought you were asking about a type with the literal name `'uint'` that some languages have.

**Reply by marcovc** [reply]
No worries, I suppose I should have explained myself better. I meant that, looking at the code it seems that cpp2 is introducing a set of type aliases abbreviating the signed/unsigned numeric types (resembling those in rust), but apparently misses "uint". I noticed because it is one that I use a lot :)

**Comment by hsutter** [comment]
Thanks! This was just an oversight, adding it now...

**Comment by hsutter** [comment]
Resolved with this commit: 7da4ff7e88d8ace5bbd16459b4f46ad2387f8107

Thanks again.
